server/middlewares/loginValidation.js

```diff
@@ -0,0 +1,12 @@
+const { loginSchema } = require('../utils/validations');
+// login input Validation
+const loginValidation = (req, res, next) => {
+  loginSchema.validateAsync(req.body)
+    .then(() => {
+      next();
+    }).catch((err) => {
+      res.json({ message: err.message });
```

Comment:
As you're not specifying your status code it will be 200, and this is not the correct one.